Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM microsoft/mssql-server-linux:2017-GA
+FROM mcr.microsoft.com/mssql/server:2017-latest
```

Comment:
Hi @stronk7,
I'd stick w/ a known version to be more "conservative" - read: to get a consistent environment - when there will be the need to rebuild the image i.e. `2017-CU18-ubuntu-16.04` instead of `latest`, unless you've decided that any new build should pull a new `latest`.

HTH,
Matteo

Reply:
Yeah, i was thinking about that for some time, really was not 100% sold to any option. One side, sticky guarantees it will continue working on rebuild. Other side, latest makes possible to early detect problems with new versions (like the 2019 case).

Also was considering if, maybe, we could go build both the sticky and the latest (and also test more both). Problem is that, right now, there are 2 "latest" ones, the 2017 and the 2019... 

So really I'm undecided here (it also weights that we rarely rebuild these images...).

Hate choices, lead to frustration because no choice is perfect, heh :-)

Reply:
Have changed it to `mcr.microsoft.com/mssql/server:2017-CU18-ubuntu-16.04` after all!